CHANGELOG.rst

```diff
@@ -78,6 +78,9 @@ Changed
 Removed
 ~~~~~~~
 
+* Removed ``Mistral``
```

Comment:
Minor recommendation: 
```suggestion
* Removed ``Mistral`` workflow engine (deprecation) #5011
```

at least, having a PR # number in the changelog is helpful.